core/genesis.go

```diff
@@ -415,6 +415,7 @@ func DefaultCalaverasGenesisBlock() *Genesis {
 func DeveloperGenesisBlock(period uint64, faucet common.Address) *Genesis {
 	// Override the default period to the user requested one
 	config := *params.AllCliqueProtocolChanges
+	config.Clique = config.Clique.Copy()
```

Comment:
There is no need to add a `Copy` method here, just copy the value through

```
config.Clique = &params.CliqueConfig{
    Period: period,
    Epoch: config.Clique.Epoch,
}
```

Reply:
Ok that makes it much easier :see_no_evil: